### PR TITLE
BR, TiCDC: update br, cdc's docs and FAQ

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update br, cdc's documents and FAQ.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5450
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -347,6 +347,35 @@ Since v4.0.8, if the `canal` or `maxwell` protocol is used for output in a chang
...
+
+## When using TiCDC to create a replication task, set the `enable-old-value` to `true`, the upstream `INSERT`/`UPDATE` statement becomes `REPLACE INTO` after being replicated to the downstream by TiCDC
+
+TiCDC defaults to specifying the `safe-mode` to `true` when creating a changefeed, thereby generating `REPLACE INTO` execution statements for the upstream `INSERT`/`UPDATE` statements.
+
+Currently, users can not modify the `safe-mode` setting, thus this issue has no solution.
+
+## When using TiCDC to replicate messages to Kafka, Kafka returns the `Message was too large` error
+
+For TiCDC v4.0.8 or earlier versions, only configuring the `max-message-bytes` for Kafka in the Sink URI does not effectively control the size of the message output to Kafka. To increase the byte limit of Kafka receiving messages, you need to add the following configuration to the Kafka server configuration.
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:368]:
```suggestion
For TiCDC v4.0.8 or earlier versions, you cannot effectively control the size of the message output to Kafka only by configuring the `max-message-bytes` setting for Kafka in the Sink URI. To control the message size, you also need to increase the limit on the bytes of messages to be received by Kafka. To add such a limit, add the following configuration to the Kafka server configuration.
```

File: ticdc/troubleshoot-ticdc.md [hunk 2]
```diff
@@ -347,6 +347,35 @@ Since v4.0.8, if the `canal` or `maxwell` protocol is used for output in a chang
...
+
+```
+# The maximum bytes number of a message that the broker receives
+message.max.bytes=2147483648
+# The maximum bytes number of a message that the broker copies
+replica.fetch.max.bytes=2147483648
+# The maximum message bytes number that the consumer side reads
+fetch.message.max.bytes=2147483648
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:376]:
```suggestion
# The maximum byte number of a message that the broker receives
message.max.bytes=2147483648
# The maximum byte number of a message that the broker copies
replica.fetch.max.bytes=2147483648
# The maximum message byte number that the consumer side reads
fetch.message.max.bytes=2147483648
```

File: br/backup-and-restore-faq.md [hunk 3]
```diff
@@ -85,3 +85,11 @@ No. BR does not support in-place full recovery of some historical backup.
 ## How can I use BR for incremental backup in the Kubernetes environment?
 
 To get the `commitTs` field of the last BR backup, run the `kubectl -n ${namespace} get bk ${name}` command using kubectl. You can use the content of this field as `--lastbackupts`.
+
+## After BR restores the archive, do I need to execute the `ANALYZE` statement in the table to update the statistics left by TiDB on the table and index?
+
+BR does not back up statistics (except v4.0.9). Therefore, after restoring the archive, you need to manually execute `ANALYZE TABLE` or wait for TiDB to automatically execute `ANALYZE`.
+
+BR v4.0.9 backup statistics makes BR consume too much memory. To ensure the normal backup process, the backup statistics feature is disabled by default starting from v4.0.10.
```
[the remark this answers is not in the document]
Comment (lichunzhu) [reply br/backup-and-restore-faq.md:93]: LGTM

File: br/backup-and-restore-faq.md [hunk 4]
```diff
@@ -85,3 +85,11 @@ No. BR does not support in-place full recovery of some historical backup.
 ## How can I use BR for incremental backup in the Kubernetes environment?
 
 To get the `commitTs` field of the last BR backup, run the `kubectl -n ${namespace} get bk ${name}` command using kubectl. You can use the content of this field as `--lastbackupts`.
+
+## After BR restores the archive, do I need to execute the `ANALYZE` statement in the table to update the statistics left by TiDB on the table and index?
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:89]:
```suggestion
## After BR restores the backup data, do I need to execute the `ANALYZE` statement on the table to update the statistics left by TiDB on the table and index?
```

File: br/backup-and-restore-faq.md [hunk 5]
```diff
@@ -85,3 +85,11 @@ No. BR does not support in-place full recovery of some historical backup.
...
+
+BR does not back up statistics (except for v4.0.9). Therefore, after restoring the backup data, you need to manually execute `ANALYZE TABLE` or wait for TiDB to automatically execute `ANALYZE`.
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:91]:
```suggestion
BR does not back up statistics (except in v4.0.9). Therefore, after restoring the backup data, you need to manually execute `ANALYZE TABLE` or wait for TiDB to automatically execute `ANALYZE`.
```

File: br/backup-and-restore-faq.md [hunk 6]
```diff
@@ -85,3 +85,11 @@ No. BR does not support in-place full recovery of some historical backup.
...
+
+BR v4.0.9 backup statistics makes BR consume too much memory. To ensure the normal backup process, the backup statistics feature is disabled by default starting from v4.0.10.
+
+If you do not execute `ANALYZE` on the table, TiDB fails to select the most optimized execution plan due to inaccurate statistics. If query performance is not a key concern, you can ignore `ANALYZE`.
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:95]:
```suggestion
If you do not execute `ANALYZE` on the table, TiDB fails to select the optimized execution plan due to inaccurate statistics. If query performance is not a key concern, you can ignore `ANALYZE`.
```

File: ticdc/troubleshoot-ticdc.md [hunk 7]
```diff
@@ -347,6 +347,35 @@ Since v4.0.8, if the `canal` or `maxwell` protocol is used for output in a chang
...
 
+## The `[tikv:9006]GC life time is shorter than transaction duration, transaction starts at xx, GC safe point is yy` error is reported when I use TiCDC to create a changefeed
+
+Solution: You need to execute the `pd-ctl service-gc-safepoint --pd <pd-addrs>` command to query the current GC safepoint and service GC safepoint. If the GC safepoint is smaller than the `start-ts` of the TiCDC replication task (changefeed), you can directly add the `--disable-gc-check` option to the `cdc cli create changefeed` command to create a changefeed.
+
+If the result of `pd-ctl service-gc-safepoint --pd <pd-addrs>` does not have `gc_worker service_id`:
+
+- If your PD version is v4.0.8 or earlier, refer to [PD issue #3128](https://github.com/tikv/pd/issues/3128) for details.
+- If your PD is upgraded from v4.0.8 or an earlier version to a later version, refer to [PD issue #3366](https://github.com/tikv/pd/issues/3366) for details.
+- For other situations, report the execution result of the above command to [AskTUG](https://asktug.com/tags/ticdc).
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:358]: @lichunzhu Any alternative to AskTUG is English version? AskTuG is only for Chinese.

Comment (lichunzhu) [reply ticdc/troubleshoot-ticdc.md:362]:
```suggestion
- For other situations, report the execution result of the above command to [Slack](https://slack.tidb.io/invite?team=tidb-community&channel=sig-migrate&ref=pingcap-community)
```

File: ticdc/troubleshoot-ticdc.md [hunk 8]
```diff
@@ -351,6 +351,35 @@ Since v4.0.8, if the `canal` or `maxwell` protocol is used for output in a chang
...
 
+## The `[tikv:9006]GC life time is shorter than transaction duration, transaction starts at xx, GC safe point is yy` error is reported when I use TiCDC to create a changefeed
+
+Solution: You need to execute the `pd-ctl service-gc-safepoint --pd <pd-addrs>` command to query the current GC safepoint and service GC safepoint. If the GC safepoint is smaller than the `start-ts` of the TiCDC replication task (changefeed), you can directly add the `--disable-gc-check` option to the `cdc cli create changefeed` command to create a changefeed.
+
+If the result of `pd-ctl service-gc-safepoint --pd <pd-addrs>` does not have `gc_worker service_id`:
+
+- If your PD version is v4.0.8 or earlier, refer to [PD issue #3128](https://github.com/tikv/pd/issues/3128) for details.
+- If your PD is upgraded from v4.0.8 or an earlier version to a later version, refer to [PD issue #3366](https://github.com/tikv/pd/issues/3366) for details.
+- For other situations, report the execution result of the above command to [Slack](https://slack.tidb.io/invite?team=tidb-community&channel=sig-migrate&ref=pingcap-community)
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:362]:
```suggestion
- For other situations, report the execution result of the above command to our [Slack channel](https://slack.tidb.io/invite?team=tidb-community&channel=sig-migrate&ref=pingcap-community).
```

**Review by lichunzhu — COMMENTED** [review]
Rest LGTM

**Pull request merged**